lib/api/route/messages.dart

```diff
@@ -193,8 +193,8 @@ Future<SendMessageResult> sendMessage(
         'to': destination.userIds,
       }}),
     'content': RawParameter(content),
-    if (queueId != null) 'queue_id': queueId, // TODO should this use RawParameter?
-    if (localId != null) 'local_id': localId, // TODO should this use RawParameter?
+    if (queueId != null) 'queue_id': RawParameter(queueId),
+    if (localId != null) 'local_id': RawParameter(localId),
```

Comment:
The change to `localId` here on this route is now totally parallel to the change to `queueId`, so probably best done in the same commit.

Then the change adding `localMessageId` on the event can remain in its own subsequent commit.